tests/unit/test_dataframe.py

```diff
@@ -892,110 +892,6 @@ def test_process_over_kwargs_error(test_scm_df):
         test_scm_df.process_over("scenario", "mean", junk=4)
 
 
-@pytest.mark.parametrize(
-    "tfilter",
-    [
-        ({"time": [dt.datetime(y, 1, 1, 0, 0, 0) for y in range(2005, 2011)]}),
-        ({"year": range(2005, 2011)}),
-        # # dropped month and day support for now...
-        # ({"month": [1, 2, 3]}),
-        # ({"day": [1, 2, 3]}),
-    ],
-)
-def test_relative_to_ref_period_mean(test_processing_scm_df, tfilter):
```

Comment:
Is this test still in ScmRun? Why is it deleted here?

Reply:
That test is still in test_run and `test_processing_scm_df` iterates over `ScmRun` and `ScmDataFrame`